domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/SearchIssuesExecutorTest.kt

```diff
@@ -0,0 +1,78 @@
+package jp.co.yumemi.android.code_check.domain.usecases
+
+import io.kotest.matchers.shouldBe
```

Comment:
<table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/91e972b3777e6ce6988574fb31ac895b140fe6d3/domain/src/test/kotlin/jp/co/yumemi/android/code_check/domain/usecases/SearchIssuesExecutorTest.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>